src/index.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable */
+import showMovies from './showMovie.js';
```

Comment:
- [ ] Kindly enable the linter for this file and remove all linter disables. Linters enforces code quality and therefore should not be disabled. Instead, you should correct the issues raised.

Reply:
Addressed 👍🏼